tools/lib/h5tools.h

```diff
@@ -569,6 +569,7 @@ typedef struct h5tools_vfd_info_t {
 
     /* Pointer to information to be passed to the driver for its setup */
     const void *info;
+    const char *token;
```

Comment:
Since this new token is currently specific to the ROS3 VFD, it'd be much better from a maintenance perspective if this field was wrapped in a new structure that encloses both the old `H5FD_ros3_fapl_t` structure and this field so that it doesn't get mixed in with other VFDs.